pypeit/coadd1d.py

```diff
@@ -52,6 +52,7 @@ class OneSpec(datamodel.DataContainer):
         'obj_model': dict(otype=np.ndarray, atype=np.floating, desc='Object model for tellurics'),
         'ext_mode': dict(otype=str, desc='Extraction mode (options: BOX, OPT)'),
         'fluxed': dict(otype=bool, desc='Fluxed?'),
+        'spect_meta': dict(otype=dict, desc='header dict'),
```

Comment:
I'm confused. This change was from Feige Wangs last tellfit_script brach? Why is it here? Did something go wrong with the merge?